notebook/static/notebook/js/notebook.js

```diff
@@ -401,7 +401,7 @@ define(function (require) {
         var st = sme.scrollTop();
         var t = sme.offset().top;
         var ct = cells[index].element.offset().top;
-        var scroll_value =  st + ct - (t + .01 * percent * h);
+        var scroll_value =  st + ct - (t + 0.01 * percent * h);
```

Comment:
Those already existed, I just added a 0 in front because `0.01` looks better than `.01` (my linter agrees).